NEW_PROJECT_APPLICATION.md

```diff
@@ -31,6 +31,8 @@ For each repository, please provide:
 * Who are the members of the leadership team?
 * How are members of the leadership team nominated?
 * How are individuals outside of leadership given commit access?
+* Is the project open/willing to move to more open goverance (if not already in place)?
+* Is the project willing to adopt the OpenJS Code of Conduct (CoC)?
```

Comment:
Should these two changes be broken out into a separate update? They are reasonable requests, but they are someone tangential to the bulk of this PR which is about roles for proposal process

Reply:
@Myles I can, but it is worth it?

Reply:
maybe not if we have consensus.

Reply:
```suggestion
* Is the project willing to adopt the OpenJS Foundation Code of Conduct (CoC)?
```